microconfig-cli/src/main/java/io/microconfig/MicroconfigMain.java

```diff
@@ -39,21 +43,22 @@ public static void main(String... args) {
         }
 
         File rootDir = params.rootDir();
-        File destinationDir = params.destinationDir();
-        String env = params.env();
+        String destinationDir = params.destinationDir();
+        List<String> envs = params.envs();
         List<String> groups = params.groups();
         List<String> services = params.services();
         boolean stacktrace = params.stacktrace();
         boolean jsonOutput = params.jsonOutput();
 
-        new MicroconfigMain(rootDir, destinationDir, env, groups, services, stacktrace, jsonOutput).build();
+        new MicroconfigMain(rootDir, destinationDir, envs, groups, services, stacktrace, jsonOutput).build();
     }
 
     private void build() {
         try {
-            long startTime = currentTimeMillis();
-            doBuild();
-            announce("\nGenerated [" + env + "] configs in " + (currentTimeMillis() - startTime) + "ms");
+            enableLogger(!jsonOutput);
+            doBuild(envs.contains("*")
+                    ? new MicroconfigRunner(rootDir, null).getMicroconfig().environments().environmentNames()
```

Comment:
would be nice if this was able to be grabbed without instantiating the runner, but i didnt want to mess with too much stuff !